### PR TITLE
Add support for semantic versions

### PR DESCRIPTION
**Pull request by romanmatyus**
If the minimum version of php was increased to `8.0` in `4.0`, it would not be necessary to modify `int` to `string` in the API settings.

### CODE REVIEW

File: src/EndpointIdentifier.php [hunk 1]
```diff
@@ -49,4 +54,9 @@ public function getUrl(): string
     {
         return "v{$this->version}/{$this->package}/{$this->apiAction}";
     }
+
+    private function checkVersionFormat(string $version): bool
```
Comment (lulco) [inline src/EndpointIdentifier.php:58]: toto by som necheckoval, mozu si ludia chciet nechat tie verzie ako boli 1, 2, 3, atd. pripadne si tam dat akykolvek iny string ak chcu, nemusia to byt cisla

**Review by lulco — CHANGES_REQUESTED** [review]

File: src/EndpointIdentifier.php (hunk 1 above)
Comment (romanmatyus) [reply src/EndpointIdentifier.php]: pravda, ale ponechal som kontrolu aspon `/` kvoli routovaniu.

**Comment by lulco** [comment]
👍 Please rebase.

**Comment by romanmatyus** [comment]
Hotovo

**Pull request merged**